src/parser/jobs/dnc/modules/DirtyDancing.tsx

```diff
@@ -268,7 +268,7 @@ export class DirtyDancing extends Analyser {
 			target: 95,
 			requirements: [
 				new Requirement({
-					name: <Fragment><StatusLink {...this.data.statuses.STANDARD_FINISH} /> uptime</Fragment>,
+					name: <Fragment><StatusLink {...this.data.statuses.STANDARD_FINISH} /> <Trans id="dnc.dirty-dancing.checklist.standard-finish-buff.uptime"> uptime</Trans></Fragment>,
```

Comment:
I feel link instead of <Fragment><StatusLink/><Trans /></Fragment>, this probably ought to just be <Trans><StatusLink /></Trans>. Pretty sure the way it is now violates some kind of i18n convention.

Reply:
Resolved by 7c45853